### PR TITLE
Ignore sysctl errors

### PR DESCRIPTION
**Pull request by fmarier**

### CODE REVIEW

File: deb/brave-keyring/DEBIAN/postinst [hunk 1]
```diff
@@ -6,5 +6,5 @@ chmod 0644 /etc/apt/trusted.gpg.d/brave-browser-release.gpg
 # Delete (expired) key added by old install instructions
 apt-key --keyring /etc/apt/trusted.gpg del D8BAD4DE7EE17AF52A834B2D0BB75829C2D4E821
 
-# Enable user namespaces immediately
-sysctl -p /etc/sysctl.d/brave.conf
+# Enable user namespaces immediately if they're available
+sysctl -p /etc/sysctl.d/brave.conf || true
```
Comment (keur) [inline deb/brave-keyring/DEBIAN/postinst]: @fmarier The more idiomatic way of doing this would be using `sysctl -p (--ignore)`.

Comment (fmarier) [reply deb/brave-keyring/DEBIAN/postinst]: Ah nice. Yes, I should have done that! Thanks @keur.

Comment (bkero) [reply deb/brave-keyring/DEBIAN/postinst]: Thanks keur. Will get this added during the next release.

Comment (bkero) [reply deb/brave-keyring/DEBIAN/postinst]: Fixed in https://github.com/brave/brave-release/pull/12

**Pull request merged**